### ISSUE TITLE
Plans for Flask-Talisman

### ISSUE THREAD

**Issue by collinsmarra** [GoogleCloudPlatform/flask-talisman#81]
what are the plans for flask-talisman? I see the last contributions were made 3 years ago

**Comment by Jonakemon** [comment]
Hi @collinsmarra, we've actively maintained the package in a new repository: https://github.com/wntrblm/flask-talisman

**Comment by jelly** [comment]
Is that fork blessed by the folks who wrote `flask-talisman`? If so can this repository get archived because on PyPi the homepage now points to fork.

**Comment by theacodes** [comment]
Yup. That's me. I wrote the project while at google, I forked it after I left. No one at big G has touched it since.

**Comment by Jonakemon** [comment]
Would be great if this repository could be archived. @theacodes do you know an admin of the GoogleCloudPlatform organization? If not, I can see if I can reach someone at Google Open Source.
